web/admin/src/request/types.ts

```diff
@@ -1381,7 +1384,8 @@ export interface V1RssParseResp {
 }
 
 export interface V1RssScrapeReq {
-  id?: string;
+  id: string;
```

Comment:
在 `V1RssScrapeReq` 接口中，`id` 字段从可选变为了必填。请确认这是否符合后端接口的实际要求，如果后端接口仍然接受可选字段，则此处修改可能会导致不必要的限制。

```suggestion
// 如果后端接口支持可选字段，请保持一致性
id?: string;